app/helpers/application_helper/button/volume_attach.rb

```diff
@@ -1,10 +1,6 @@
 class ApplicationHelper::Button::VolumeAttach < ApplicationHelper::Button::Basic
-  def calculate_properties
-    super
-    self[:title] = @record.is_available_now_error_message(:attach_volume) if disabled?
-  end
-
   def disabled?
-    !@record.is_available?(:attach_volume)
+    @error_message = @record.is_available_now_error_message(:attach_volume) unless @record.is_available?(:attach_volume)
```

Comment:
[GenericFeatureButtonWithDisable](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/application_helper/button/generic_feature_button_with_disable.rb) can be used here